source/documentation/logging/logit-joiners.md

```diff
@@ -4,10 +4,10 @@ If you do not have a Logit account you can create one by signing into Logit for
 
 To sign into Logit:
 
-1. Log into any G Suite app, such as GDS GMail or Calendar.
-1. Click on the 9 small dots at the top right of the page.
-1. click **More** at the bottom to show more options.
-1. scroll down and click on the **Logit** icon.
+1. Go to https://logit.io/ and click Sign In
```

Comment:
You can save a click (at the moment) by going straight to https://logit.io/dashboard.

This will also remember the email you logged in with last (once you've done the other steps once).